src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -2015,3 +2026,171 @@ def _test_backups_mgmt(self, database_engine, resource_group, server):
 
             self.assertEqual(backup_name, customer_backup['name'])
             self.assertDictEqual(customer_backup, backups[0])
+
+
+class FlexibleServerIdentityAADAdminMgmtScenarioTest(ScenarioTest):
+    mysql_location = 'northeurope'
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location=mysql_location)
+    def test_mysql_flexible_server_identity_aad_admin_mgmt(self, resource_group):
+        self._test_identity_aad_admin_mgmt('mysql', resource_group)
+
+    def _test_identity_aad_admin_mgmt(self, database_engine, resource_group):
+        login = 'alanenriqueo@microsoft.com'
+        sid = '894ef8da-7971-4f68-972c-f561441eb329'
+
```

Comment:
Not sure if it's okay to add this here. Need second eye from CLI team for this

Reply:
There is a [similar test](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py#L850) for single server.

Reply:
My suggestion is to create a test user like: https://github.com/Azure/azure-cli/blob/79d3e4d4cab6830ecd6cb4a142895a353f3a9391/src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py#L870